README.md

```diff
@@ -116,3 +116,22 @@ To opt out of tracking, please go to the raw `.ipynb` files and remove the follo
     "![Impressions](https://PixelServer20190423114238.azurewebsites.net/api/impressions/ComputerVision/classification/notebooks/21_deployment_on_azure_container_instances.png)"
 ```
 
+
+## Install this repository via PIP
```

Comment:
Just adding this section to the end seems the wrong place. Maybe consider moving to the end of the "getting started" instructions, and condense all the text here into 3-4 sentences including when one would might want to do "pip install git+https://github.com/microsoft/ComputerVision#egg=utils_cv"

Reply:
Done in the new commit.